lib/graphql/transaction.ts

```diff
@@ -159,6 +173,46 @@ export const UPDATE_SPLIT_TRANSACTION = `mutation updateTransactionSplits(
   }
 }`;
 
+export const CREATE_TRANSACTION_ASSET = `mutation createTransactionAsset(
+  $transactionId: ID!
+  $name: String!
+  $filetype: String!
+) {
+  createTransactionAsset(
+    transactionId: $transactionId
+    name: $name
+    filetype: $filetype
+  ) {
+    assetId
+    url
+    formData {
+      key
+      value
+    }
+  }
+}`;
+
+export const FINALISE_TRANSACTION_ASSET = `mutation finaliseTransactionAssetUpload(
```

Comment:
In general, we try to stick with American English variant, so please use `finalize` (US) instead of `finalise` (UK). Added a section in Notion to make this clear for the future.

Reply:
I see - is this a blocker? Only because it requires a backend PR to alter it there also.